.circleci/config.yml

```diff
@@ -50,6 +50,7 @@ jobs:
       - run:
           name: "Run Python Tests"
           command: |
+            pytest --cache-clear demo/tests/
```

Comment:
Decided to include demo tests in the ci. They are good integration tests plus, they will help keep docs up to date should something break.